code/game/turfs/change_turf.dm

```diff
@@ -164,10 +164,19 @@ GLOBAL_LIST_INIT(blacklisted_automated_baseturfs, typecacheof(list(
 			QDEL_NULL(stashed_air)
 			return
 		var/turf/open/newTurf = .
+		var/turf_fire_ref
+		if(turf_fire)
+			if(isgroundlessturf(path))
```

Comment:
isgroundlessturf is for instances, not paths. you dont need to use paths after the turf has been created

Reply:
i reverted this to how it was before

Reply:
see below